src/components/abstract/resource-form/client.tsx

```diff
@@ -176,28 +177,33 @@ export function AbstractResourceFormInternal<T extends Resource>({
                 ))}
 
                 {/* // TODO: rich text editor */}
-                {richTextInputs.map((input) => (
+                {richTextInput != null && (
                   <FormField
-                    key={input.name}
+                    key={richTextInput.name}
                     control={form.control}
-                    name={input.name}
+                    name={richTextInput.name}
                     render={({ field }) => (
                       <FormItem>
-                        <FormLabel>{input.label}</FormLabel>
+                        <FormLabel>{richTextInput.label}</FormLabel>
                         <FormControl>
-                          <Input
-                            className="bg-background placeholder:text-foreground h-20 shadow-none"
-                            {...field}
-                            // TODO: figure out why field.value is a union of all possible input types
-                            // these casts should not be necessary since AbstractResourceFormInputs specifies only the keys which have the correct type
-                            value={(field.value ?? "") as string}
+                          <MinimalTiptapEditor
+                            // @ts-expect-error types not matching
+                            value={field.value ?? ""}
+                            onChange={field.onChange}
+                            className="w-full"
+                            editorContentClassName="p-5"
+                            output="html"
+                            placeholder="Enter your description..."
+                            autofocus={true}
+                            editable={true}
```

Comment:
dla propsów booleanowych wystarczy podać nazwę, domyślnie jest wtedy wartość `true`


```suggestion
                            autofocus
                            editable
```